packages/esm-ugandaemr-app/src/dashboard.meta.tsx

```diff
@@ -4,3 +4,19 @@
 //   config: { columns: 1, type: "grid" },
 //   title: "Test",
 // };
+
+export const facilityListMeta = {
+  name: 'facility-lists',
```

Comment:
Is this a work in progress cause this isn't being used anywhere

Reply:
its work in progress